collections/Laud_Misc/MS_Laud_Misc_636.xml

```diff
@@ -37,43 +59,195 @@
                      <idno type="SCN">1003</idno>
                   </altIdentifier>
                </msIdentifier>
+               <head>The Peterborough Chronicle (Anglo-Saxon Chronicle, MS. E); Peterborough, c. 1121-31, c. 1155, and s. xiii/xiv</head>
```

Comment:
I believe *Anglo-Saxon Chronicle* is usually treated as a title, unless scholarship has followed the same direction as *South English Legendary*/South English legendaries?